requirements.txt

```diff
@@ -5,6 +5,8 @@ evelink
 dnspython
 passlib
 requests>=2.9.1
+requests_cache
+json
```

Comment:
Hmm, odd that you don't find. And really, the code will should work without the package installed. The EveWhoManer relies upon it.